extensions/arc/package.json

```diff
@@ -766,12 +809,14 @@
                       "fields": [
                         {
                           "label": "%arc.postgres.server.group.cores.request%",
+                          "description": "%arc.postgres.server.group.cores.description%",
                           "variableName": "AZDATA_NB_VAR_POSTGRES_SERVER_GROUP_CORES_REQUEST",
                           "type": "number",
                           "min": 0
```

Comment:
Let's have a minimum of 1 for the cores limit and request.  That matches the validation in azdata:

```
azdata arc postgres server create -n pg -cr 0
Invalid kubernetes resource format 0. Valid Cpu resource quantities are strictly positive...

azdata arc postgres server create -n pg -cl 0
Invalid kubernetes resource format 0. Valid Cpu resource quantities are strictly positive...

```

Reply:
Sure thing thank you for showing the validation statements